### DISCUSSION TITLE
GitHub Tutorials

### DISCUSSION THREAD

**Discussion by UG-SEP** [MakeContributions/DSA#383] [Show and tell]
As a beginner, It is difficult to learn how to create Multiple PR, Contribute projects, Opening Issue, Creating repo
So I discussion you can ask about any doubt related to GitHub.
 
[How to create Multiple Pull Request](https://www.youtube.com/watch?v=aDyIjBb8z0w&t=2s)
[Tips and Tricks for GitHub users](https://www.youtube.com/watch?v=uISaVQafY88)
[Search project to contribute](https://www.youtube.com/watch?v=2B1VUo22hGA)

**Comment by ptarafdar001** [comment]
Thanks bro...yeah sure I will ask if I stack any point and I will definitely contribute more in this project.

**Reply by UG-SEP** [reply]
> Thanks bro...yeah sure I will ask if I stack any point and I will definitely contribute more in this project.

why not I am also open to help you
